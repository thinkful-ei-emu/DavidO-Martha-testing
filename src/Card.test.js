import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';
import renderer from 'react-test-renderer';

describe('<Card />', () =>{
  it('renders without crashing', ()=> {
    const div = document.createElement('div');
  
    ReactDOM.render(<Card key="1" title="Card1" content="Content of card" />, div);
  
    ReactDOM.unmountComponentAtNode(div);

  });
});

it('renders the UI as expected', () => {
  const snapshot = renderer
    .create(<Card key="1" title="Card1" content="Content of card"/>)
    .toJSON();
  expect(snapshot).toMatchSnapshot();  
});
