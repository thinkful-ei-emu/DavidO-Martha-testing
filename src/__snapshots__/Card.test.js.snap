// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Card1
  </h3>
  <p>
    Content of card
  </p>
</div>
`;
